src/locales/default.json

```diff
@@ -258,8 +258,16 @@
         "1": "Votes can be changed while the proposal is active"
       }
     },
-    "downloadCsvVotes": "Download as CSV",
-    "preparingCsvVotes": "Preparing file"
+    "downloadCsvVotes": {
+      "title": "Download as CSV",
+      "postDownloadModal": {
+        "title": "Download votes report",
+        "message": {
+          "pendingGeneration": "The report is generating… Please try again in a few minutes.",
+          "unknownError": "Unable to contact the download server. Please try again later."
```

Comment:
```suggestion
          "pendingGeneration": "Your report is currently being generated. It may take a few minutes. Please check back shortly.",
          "unknownError": "We're having trouble connecting to the server responsible for downloads. Please try again in a few moments. If the problem persists, consider contacting our support team."
```